MekHQ/src/mekhq/campaign/force/ForceType.java

```diff
@@ -20,56 +20,118 @@
+     * Convoy force type, typically used by the Resupply module.
      */
-    CONVOY("Convoy"),
+    CONVOY(true, true),
 
     /**
-     * Security force type, typically used for protection and guarding operations.
+     * Security force type, typically used by the Prisoner Events module.
      */
-    SECURITY("Security");
+    SECURITY(true, true);
 
+    // region Fields
+    private final boolean standardizeParents;
+    private final boolean childrenInherit;
 
-    // Fields
-    private final String name;
+    // region Constructor
 
-    // Constructor
     /**
-     * Constructs a {@code ForceType} with a specified name.
+     * Constructor for the {@code ForceType} enum.
      *
-     * @param name the name of the force type, used for displaying or referencing.
+     * @param standardizeParents  Whether changing to this ForceType changes the ForceType in all
+     *                           parent forces to STANDARD
+     * @param childrenInherit     Whether changing to this ForceType changes the ForceType in all
+     *                           child forces to this ForceType.
      */
-    ForceType(String name) {
-        this.name = name;
+    ForceType(boolean standardizeParents, boolean childrenInherit) {
+        this.standardizeParents = standardizeParents;
+        this.childrenInherit = childrenInherit;
     }
+    // endregion Constructor
 
 
     // region Getters
     /**
-     * Returns the name of this force type.
+     * Retrieves the display name for the ForceType by fetching a localized label from the relevant
+     * resource bundle.
+     *
+     * <p>The method uses the {@code name} of the current instance to construct a resource
+     * key in the format {@code [name].label}. This key is used to look up a localized string
+     * from the {@code ForceType} resource bundle located in the {@code mekhq.resources} package.
+     * The formatted text at the specified key is returned as the display name.</p>
+     *
+     * @return The localized display name for the current instance.
+     */
+    public String getDisplayName() {
+        final String RESOURCE_BUNDLE = "mekhq.resources.ForceType";
+        final String RESOURCE_KEY = name() + ".label";
+
+        return getFormattedTextAt(RESOURCE_BUNDLE, RESOURCE_KEY);
+    }
+
+    /**
+     * Retrieves the symbol associated with this ForceType.
+     *
+     * <p>The method determines the symbol to display for the current instance by looking up
+     * a localization resource key in the {@code ForceType} resource bundle, with keys formatted
+     * as {@code [enumName].symbol}.</p>
+     *
+     * <p>If the current instance is {@code STANDARD}, an empty string is returned as the symbol.</p>
+     *
+     * @return The localized symbol associated with the current instance, or an empty string
+     *         if the instance is {@code STANDARD}.
+     */
+    public String getSymbol() {
+        if (this == STANDARD) {
+            return "";
+        }
+
```

Comment:
Within the resource bundle just have this set to a blank string and remove the check. No reason to do an edge case when a single line can do the same thing.

Also allows for adding a symbol later if desired either in current localization or in another.